### ISSUE TITLE
Support for SwiftUI

### ISSUE THREAD

**Issue by ngochiencse** [WenchaoD/FSPagerView#315]

**Comment by ngochiencse** [comment]
@sathamkhussain No, I have to write my own SwiftUI wrapper for FSPagerView in the mean time. You can check this repo: https://github.com/ngochiencse/FSPagerViewExample-SwiftUI. For more detail check my post: https://hienphamit.com/implement-page-view-in-swiftui/

**Comment by Tarikharda** [comment]
@ngochiencse Can you give me how to transform swipe in SwiftUI because i can found to chose the type of FSPagerViewTransformer

like : pagerView.transformer = FSPagerViewTransformer(type: .overlap)

**Comment by ngochiencse** [comment]
I have not implemented this feature yet. I'll notice you when done

**Comment by iletai** [comment]
With SwiftUI, many libraries should be complete with a convert from UIKit by themself.
UIViewController -> UIViewControllerRepresentable
UIView -> UIViewRepresentable
--
I tried out that and it's really not too difficult.
But we knew Swiftui and uikit bridge were still not yet fluent. It would be best if you built a calendar by pure  @Swiftui or accepted cases of many bug-related layouts/navigation when using the library from Object-C/UIKit to SwiftUI.
